plasma/cli/client.py

```diff
@@ -0,0 +1,128 @@
+            print("Please enter a valid command ('or enter help')")
+        else:
+            return self.client_cmds[command]()
+
+    def sync_child_chain(self):
+        self.current_block = self.client.get_current_block_num()
+        while self.synced_block < self.current_block:
+            block_number = self.synced_block
+            block = self.client.get_block(self.synced_block)
+            self.db.put(bytes(block_number), utils.str_to_bytes(block))
+            print("Synced %s" % self.synced_block)
+            self.synced_block += 1
+        print("Syncing complete!")
+
+    def deposit(self):
+        if len(self.inp) != 3:
+            raise Exception("Wrong number of inputs for deposit")
+        amount1 = int(self.inp[1])
+        key = utils.normalize_key(self.inp[2])
+        newOwner1 = utils.privtoaddr(key)
+        newOwner2, amount2 = utils.normalize_address(b'\x00' * 20), 0
+        tx = Transaction(0, 0, 0, 0, 0, 0,
+                         newOwner1, amount1,
+                         newOwner2, amount2,
+                         0)
+        self.client.deposit(tx, key)
+        print("Succesfully deposited %s to %s" % (amount1, newOwner1))
+
+    def send_tx(self):
+        if len(self.inp) != 14 and len(self.inp) != 13:
+            raise Exception("Wrong number of inputs for sending a transaction!")
+        blknum1, tx_pos1, utxo_pos1 = int(self.inp[1]), int(self.inp[2]), int(self.inp[3])
+        blknum2, tx_pos2, utxo_pos2 = int(self.inp[4]), int(self.inp[5]), int(self.inp[6])
+        newowner1 = utils.normalize_address(self.inp[7])
+        amount1 = int(self.inp[8])
+        newowner2 = utils.normalize_address(self.inp[9])
+        amount2 = int(self.inp[10])
+        fee = int(self.inp[11])
+        key1 = utils.normalize_key(self.inp[12])
+        key2 = b''
+        if len(self.inp) == 14:
+            key2 = utils.normalize_key(self.inp[13])
+        tx = Transaction(blknum1, tx_pos1, utxo_pos1,
+                         blknum2, tx_pos2, utxo_pos2,
+                         newowner1, amount1,
+                         newowner2, amount2,
+                         fee)
+        tx.sign1(key1)
+        if key2:
+            tx.sign2(key2)
+        self.client.apply_transaction(tx)
+        print("Succesfully added transaction!")
+
+    def submit_block(self):
+        if len(self.inp) != 2:
+            raise("Wrong number of inputs to submit block")
+        key = utils.normalize_key(self.inp[1])
+        block = self.client.get_current_block()
+        block = rlp.decode(utils.decode_hex(block), Block)
+        block.make_mutable()
+        block.sign(key)
+        self.client.submit_block(block)
+        print("Successfully submitted a block!")
+
+    def withdraw(self):
+        blknum, txindex, oindex = int(self.inp[1]), int(self.inp[2]), int(self.inp[3])
+        txPos = [blknum, txindex, oindex]
+        key1 = utils.normalize_key(self.inp[4])
+        key2 = b''
```

Comment:
Could consider using [conditional expressions](https://www.python.org/dev/peps/pep-0308/). In this case:
`key2 = utils.normalize_key(self.inp[5]) if len(self.inp) == 6 else b''`

Depends on readability preferences